src/main/java/com/rtb/projectmanagementtool/auth/LoginServlet.java

```diff
@@ -28,9 +28,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     }
 
     // Get login URL
-    request.setAttribute("loginUrl", auth.getLoginLink(/*Return URL*/ "/home"));
+    request.setAttribute("loginUrl", auth.getLoginLink(/*Return URL*/ "/LoginServlet"));
+
+    // Get login URL for first time users (on submit returns to LoginServlet)
+    request.setAttribute(
+        "loginUrlNewUser", auth.getLoginLink(/*Return URL*/ "/create-new-user.jsp"));
 
     // Forward to login page
     request.getRequestDispatcher("login.jsp").forward(request, response);
+
+    // login the user with cookies
+    auth.loginUser(request, response);
+
+    // redirect back home
+    response.sendRedirect("/home"); // error here
```

Comment:
"Response has already been committed" error. Shouldn't this not be the case bc of the return in the IF?

Reply:
don't need this redirect anymore since we're already doing this up top.  The error is saying `getRequestDispatcher` already commited the response